config/puma.rb

```diff
@@ -0,0 +1,38 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
This file is generating a bunch of Rubocop complaints. Rather than fix them only to have `app:update` try to un-fix them later, let's add this file as an exclusion. We should add a subtask to make that exclusion in the searchgov-style gem, as we'll run into this with all the Rails apps.

Reply:
On second thought - we can make that change with any other Rubocop cleanup for SRCH-2230.